plugins/modules/mysql_user.py

```diff
@@ -22,7 +22,7 @@
     required: true
   password:
     description:
-      - Set the user's password.
+      - Set the user's password. Only for `mysql_native_password` authentication. For other authentication plugins see the combination of plugin, plugin_hash_string, plugin_auth_string.
```

Comment:
```suggestion
      - Set the user's password. Only for C(mysql_native_password) authentication. For other authentication plugins see the combination of I(plugin), I(plugin_hash_string), I(plugin_auth_string).
```
Ansible has special format macroses for things like option names, etc. :)